### DISCUSSION TITLE
how to restrict cmake find to MODULES?

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#2212] [Q&A] [answered]
I have specified using cmake -DMODULES, which modules need to be included in baresip lib.  Still cmake executes FindXXX.cmake for all other modules too.  Is there some way avoid the not needed finds?

**Comment by sreimers** [comment, marked as answer]
I think there is no easy way (since you would need a module FindXXX.cmake dependency to calculate before), but it should be harmless, since its the same for not detected a module.
